examples/PhasorDynamics/Tiny/ThreeBus/ThreeBus_Basic/Example_ThreeBus_Basic.cpp

```diff
@@ -8,7 +8,7 @@
  * compares results with data generated for the same system by Poweworld.
  *
  */
-#include "example2.hpp"
+#include "Example_ThreeBus_Basic.hpp"
 
```

Comment:
Consider dropping "Example" from the example name. I would also use camel case and drop underscores.
```suggestion
#include "ThreeBusBasic.hpp"
```

Reply:
I would suggest standardizing on camel case for all file names. In this case, we have a mix of camel case and underscores.